### PR TITLE
Adding a date to hold options

### LINKED ISSUE #4619: Daily Docket: Expand Actions dropdowns, add Hold Open date
As a judge, I would like to know the exact date that is 30, 60 90 days from the hearing date.

## AC
1. Add a date to the Hold Open dropdown options: 30, 60, 90 days. The date displayed should be 30, 60, or 90 days from the hearing date.
1. Expand the Hold Open dropdown to accommodate the Hold Open dates.
1. Expand all the Actions dropdowns to be the same width as the Hold Open dropdown. 

**Mock**
hold open

### PR DESCRIPTION
**Pull request by tejans24**
Resolves #4619 

### Description
Check the daily docket page's hold open dropdown for dates

### CODE REVIEW

**Comment by tejans24** [comment]
I wonder if we need a min-width for these drop downs. @annienguyenUSDS @lakohl 

screen shot 2018-04-03 at 12 56 16 pm

When the window is made a bit smaller:

screen shot 2018-04-03 at 12 56 29 pm

File: client/app/hearings/components/DocketHearingRow.jsx [hunk 1]
```diff
@@ -42,15 +42,17 @@ const dispositionOptions = [{ value: 'held',
...
-const holdOptions = [
-  { value: 0,
-    label: '0 days' },
-  { value: 30,
-    label: '30 days' },
-  { value: 60,
-    label: '60 days' },
-  { value: 90,
-    label: '90 days' }];
+const holdOption = (hearingDate, days) => ({
```
Comment (sharonwarner) [inline client/app/hearings/components/DocketHearingRow.jsx]: This is super nitpicky (sorry!). Can we just order the variables in the order they're displayed in the output? so 'const holdOption = (days, hearingDate) => ({'

Comment (tejans24) [reply client/app/hearings/components/DocketHearingRow.jsx]: haha sure, I'll update that

**Comment by tejans24** [comment]
screen shot 2018-04-03 at 3 15 47 pm

screen shot 2018-04-03 at 3 19 48 pm


@lakohl adding min-width and reducing window size puts the dropdown underneath.

**Comment by lakohl** [comment]
Looks good!

**Pull request merged**